mobile/library/cc/engine_builder.cc

```diff
@@ -845,6 +845,7 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
   }
 
   (*runtime_values.mutable_fields())["disallow_global_stats"].set_bool_value(true);
+  (*runtime_values.mutable_fields())["enable_dfp_dns_trace"].set_bool_value(true);
```

Comment:
i don't know if we always want this on in mobile, we should be able to turn it off in case it causes issues.  We can control the runtime guards via https://github.com/envoyproxy/envoy/blob/b203e3e826af06e3639229f816f2a11245476728/mobile/library/cc/engine_builder.cc#L324